src/internal/cluster/state.go

```diff
@@ -16,13 +16,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Zarf Cluster Constants.
 const (
 	ZarfNamespace       = "zarf"
 	ZarfStateSecretName = "zarf-state"
 	ZarfStateDataKey    = "state"
 )
 
-// LoadZarfState returns the current zarf/zarf-state secret data or an empty ZarfState
+// LoadZarfState returns the current Zarf/zarf-state secret data or an empty ZarfState.
```

Comment:
This would be a case where Zarf is lowercase, as this is mirroring info from a k8s manifest?

Reply:
Changed